pkg/manifests/manifests.go

```diff
@@ -2106,6 +2106,8 @@ func (f *Factory) MetricsServerDeployment(apiAuthSecretName string, kubeletCABun
 		"--requestheader-extra-headers-prefix="+requestheaderExtraHeadersPrefix,
 		"--requestheader-group-headers="+requestheaderGroupHeaders,
 		"--requestheader-username-headers="+requestheaderUsernameHeaders,
+		"--tls-cert-file=/etc/client-ca-bundle/tls.crt",
```

Comment:
No test was failing when using `/etc/tls/private/xxx`, do they have the same content? Is testing this difficult?

Reply:
no failure I just moved it from jsonnet to here 
https://github.com/openshift/cluster-monitoring-operator/pull/2293/files#r1557090512

Reply:
https://github.com/openshift/cluster-monitoring-operator/blob/889bb2ce19cd1018888f506dd8c401cca163cfc8/jsonnet/components/metrics-server.libsonnet#L120 (metrics-server-tls) is been mounted at /etc/tls/private (https://github.com/openshift/cluster-monitoring-operator/blob/889bb2ce19cd1018888f506dd8c401cca163cfc8/jsonnet/components/metrics-server.libsonnet#L248)

In the updated approach as well it uses the same secret https://github.com/openshift/cluster-monitoring-operator/pull/2293/files#diff-0bbbc0b6e7ee4c4efe218030b64c2ef604fda58d92e51e52c206b6c5c0a4a5a3R158

Reply:
I updated the PR to remove unused in volume mounts https://github.com/openshift/cluster-monitoring-operator/pull/2293/files#r1557090512